docs/report/report.tex

```diff
@@ -52,15 +52,92 @@ \subsubsection{Stretch Goals}
+In Aufbau und Zweck orientierten sich diese regelmäßigen Meetings an den "Daily Standups" der Scrum-Methode.
+Während den Meetings stellte jedes Teammitglied kurz vor was es seit dem letzten Standup am Projekt bearbeitet hatte, welche Probleme dabei aufgetreten waren und was bei der Arbeit gelernt wurde.
+Auch welche Themen jedes Mitglied bis zum nächsten Standup bearbeiten wollte, wurde besprochen.
+
+\subsubsection{Am Ende der Sprints}
+Nach jedem Sprint wurde das Ergebnis bzw. das Produktinkrement den Kunden und der Projektbetreuung vorgestellt.
+Hier wurde meist auch zusammen mit macio das Ausmaß des nächsten Produktinkrements besprochen.
+Die resultierenden Wünsche und Feedback wurden mit ins Backlog und die folgenden Besprechungen mitgenommen.
+Im Anschluss fand immer eine Retrospektive statt.
+In dieser besprach das Team die eigene Zusammenarbeit und den gemeinsamen Umgang.
+Das Feedback in diesem Kontext war konstruktiv und fair aber auch ehrlich.
+Die daraus entstandenen Erkenntnisse und Verbesserungsvorschläge wurden genutzt um das Teamwork im folgenden Sprint positiv zu halten und weiter zu optimieren.
+Direkt nach jeder Retrospektive startete der nächste Sprint beginnend mit dem entsprechenden Planing.
+
+\subsubsection{Open-Source}
+Teil der Anforderung war, dass die Software als Open-Source-Projekt der Öffentlichkeit zur Verfügung stehen soll.
+Dementsprechend entwickelte das Team den Anspruch, das dieses Projekt nicht nur Open-Source sondern auch erweiterbar und verständlich sein soll.
+Aufgrunddessen wurde die Entscheidung getroffen, das Projekt von Entwicklungsbeginn öffentlich auf github aufzusetzten.
+Diese Platform bot sich an, da sie für den Umfang des Projekts viele hilfreiche Funktionalitäten in den Bereichen Automatisierung und Kollaboration bietet, allerdings trotzdem kostenlos ist
+Diese Werkzeuge halfen dabei den Entwicklungsprozess sehr intuitiv und zentral zu gestalten.
+So wurden zum Beispiel die meisten Diskussionen zu fachlichen Themen direkt in den Arbeitspaketen.
+Auch im Nachhinein können dann Entscheidungen und Denkprozesse von anderen Kollaborierenden nachvollzogen werden.
+Dies hilft zusätzlich bei der zukünftigen Instandhaltung und Weiterentwicklung des Projekts.
+\\
+Das Produkt-Backlog wurde auch in dieses github repository eingepflegt.
+Dies erhöhte einerseits die Übersichtlichkeit über den Stand des Projekts innerhalb, da Arbeitspakete und zugehöriger Code sehr einfach verknüpft werden konnten.
+Andererseits ermöglicht diese Zusammenlegung auch, dass andere Entwickler:innen einfach ins Projekt einsteigen und ihre Arbeit dokumentieren können, ohne andere Plattformen aufsuchen zu müssen.
+Die Möglichkeiten der Automatisierung wurden genutzt, um den Workflow zu vereinfachen und um die Code-Qualität sicherzustellen.
+Dies geschah zum Beispiel die durch die automatische Erstellung von Branches für einzelne Arbeitspakete bei Beginn der Bearbeitung und automatische statische und dynamische Tests des Codes.
```

Comment:
```suggestion
Dies geschah zum Beispiel die durch die automatische Erstellung von Branches durch einen \textit{Bot}\footnote{https://github.com/renovatebot/renovate} für einzelne Arbeitspakete bei Beginn der Bearbeitung und automatische statische und dynamische Tests des Codes.
```

Reply:
Ich hoffe, das hat geklappt mit der Fußnote (bitte prüfen 😅 )

Reply:
Ich würde in diesem Abschnitt noch nicht auf so technische Details eingehen. Das passt später wesentlich besser.